<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphDialog</class>
 <widget class="QDialog" name="GraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Grafo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Digite Vertice</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Digite Arista</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_value">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_value2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_value3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_insert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(124, 255, 144);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string> INSERTAR VERTICE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 0, 0);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>ELIMINAR VERTICE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>891</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_insertA">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(124, 255, 144);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>INSERTAR ARISTA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regresarG">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 0);
</string>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
